server/liveblog/embed/embed.py

```diff
@@ -104,11 +105,15 @@ def embed(blog_id, api_host=None, theme=None):
     # retrieve the wanted theme and add it to blog['theme'] if is not the registered one
     try:
         theme_name = request.args.get('theme', theme)
+
     except RuntimeError:
         # this method can be called outside from a request context
         theme_name = theme
-    # collect static assets to load them in the template
+
     theme = get_resource_service('themes').find_one(req=None, name=blog['blog_preferences'].get('theme'))
+    if theme is None:
```

Comment:
you also need to check if `theme_name` is **not** `None` here, otherwise, the fall-back from parameter won't work